src/main/webapp/resources/js/pages/UserGroupsPage/components/UserGroupProjectsTable.jsx

```diff
@@ -34,9 +42,24 @@ export function UserGroupProjectsTable({ groupId }) {
     },
   ];
 
-  useEffect(() => {
-    getProjectsForUserGroup(groupId).then((data) => setProjects(data));
+  React.useEffect(() => {
+    getProjectsForUserGroup(groupId).then((data) => {
+      setProjects(data);
+      setTotal(data.length);
+    });
   }, [groupId]);
 
-  return <Table columns={columns} dataSource={projects} />;
+  return (
+    <Table
+      columns={columns}
+      dataSource={projects}
+      pagination={{
+        total: total,
```

Comment:
Since you are passing `total` to the page optins anyways, you could include it in that returned object.  then here all you would need to do is:

```javascript
pagination={paginationOptions}
```

Reply:
What would happen if `paginationOptions` has't been set yet before the first render? Wouldn't that crash the page?

Reply:
Nope, it would just be undefined, which to ant just means don't display the pagination.

Reply:
Perfect thanks. I was having other issues that was causing the page to crash but is fixed now